apps/web/pages/settings/my-account/out-of-office/index.tsx

```diff
@@ -39,18 +40,58 @@ export type BookingRedirectForm = {
   toTeamUserId: number | null;
   reasonId: number;
   notes?: string;
+  uuid?: string | null;
 };
 
-const CreateOutOfOfficeEntryModal = ({
+export type outOfOfficeEntryData = {
+  uuid: string;
+  selectedReason: number | null;
+  profileRedirect: boolean;
+  selectedMember: number | null;
+  notes: string | null;
+  start: Date;
+  end: Date;
+};
+
+const CreateOrEditOutOfOfficeEntryModal = ({
   openModal,
   closeModal,
+  currentlyEditingOutOfOfficeEntry,
```

Comment:
What’s the need to create an object and define its types? Can’t we just pass the whole item and export its type from the router output, then use it accordingly?